src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const menu = document.querySelector('.page__menu');
```

Comment:
it's great to use data-attributes in javascript or specific classes with '-js' suffix
this way you can separate the layout and scripts

idk if it's reasonable for this case because we want to toggle very specific classnames

basically I use atomic classname 'is-active' for such cases

Reply:
I want to keep it BEM that's why I used BEM classes to find elements so the logic stick to BEM name